### PR TITLE
[ENHANCEMENT] new "Contribution" page

### PR DESCRIPTION
**Pull request by shaedrich**
- Relates to #566
- Fixes #590

### CODE REVIEW

**Comment by shaedrich** [comment]
@pfreitag I fixed guides rendering as requested in https://github.com/foundeo/cfdocs/pull/590#issuecomment-338764538

File: doc.cfm [hunk 1]
```diff
@@ -2,12 +2,13 @@
 <cfset url.name = ReReplace(url.name, "[^a-zA-Z0-9_-]", "", "ALL")>
 <cfif url.name IS "index">
 	<cfset data = {name="CFDocs", description="Ultra Fast CFML Documentation", type="index"}>
-<cfelseif FileExists(ExpandPath("./guides/en/#url.name#.md"))>
+<cfelseif FileExists(ExpandPath("./guides/en/#url.name#.md")) && url.name is "how-to-contribute">
```
Comment (pfreitag) [inline doc.cfm]: This change would cause all guides to fail /except how-to-contribute right?

Comment (shaedrich) [reply doc.cfm]: Silly me. I changed it.

**Comment by shaedrich** [comment]
I made the requested changes. Can we go on?

**Pull request merged**